src/libs/highlevel/elektra.c

```diff
@@ -303,6 +306,26 @@ void elektraFatalError (Elektra * elektra, ElektraError * fatalError)
 	elektra->fatalErrorHandler (fatalError);
 }
 
+/**
+ * Generate a error message about a problem with the specification.
+ * 
+ * @note:  The returned char array needs to be freed with elektraFree() after usage.
+ * 
+ * @param application The application's name.
+ * @return Pointer to a char. Needs to be freed using elektraFree() after usage.
+ */
+static char * generateSpecProblemErrorMessage( const char * application) {
+	return elektraFormat(
+		"There was a problem with the application's specification. \n\nTo fix this, execute:\n"
+		"\t\"$ sudo kdb rm -r %s\"\n"
```

Comment:
Why do we recommend to remove everything?

Reply:
Actually, you are right: the user should only `sudo kdb rm -r spec:/sw/jonls/...` !

Reply:
Fixed!